test/spec/aggregate.test.ts

```diff
@@ -10,6 +10,7 @@ interface IAggregateSigsTestCase {
     input: string[];
     output: string;
   };
+  meta?: undefined;
```

Comment:
This is a very strange type, if they property is optional it can already be undefined. I haven't checked how this is used or why we had to add the type here but should either set a proper type, or use unknown instead

Reply:
Am also curious why this got added if its unused.  I know the spec tests have the meta field but if we are not using them in any test cases can we just "ignore" that they are there?  Thinking out loud I suppose be "more correct" is better though... I guess im done rambling now...  I suppose leaving them there is the right thing to do and we can make use of it in the future if necessary